app/javascript/packs/application-style.js

```diff
@@ -0,0 +1,7 @@
+import 'components/app/app-container.scss';
```

Comment:
I like what you've done all over.
You can consider adding css-modules (which is a hassle) but allows us to style each component without worrying about affecting the rest of the styling. (this is more of a initialization challenge than you probably a coding one, you can skip it.)

Reply:
css modules are integrated in the project :)
iv'e being able to use both global styles and css modules (in case you need css reset, bootstrap or any other global style)
i also like creating a directory for each major section with the corresponding scss, in order for this to work i created the application-style which import all the styles for webpacker to compile